tool/tsh/db.go

```diff
@@ -590,7 +590,14 @@ func getMongoCommand(tc *client.TeleportClient, profile *client.ProfileStatus, d
 	if db.Database != "" {
 		args = append(args, db.Database)
 	}
-	return exec.Command(mongoBin, args...)
+
+	// look for `mongosh`, fall back to `mongo` if not found.
+	clientPath, err := exec.LookPath(mongoshBin)
+	if err != nil {
+		return exec.Command(mongoBin, args...)
+	}
+
+	return exec.Command(clientPath, args...)
```

Comment:
Yes, they are.

One thing we may want to do with respect to CLI arguments is to switch from old names (`--ssl`, `--sslPEMKeyFile`, `--sslCAFile`) to their modern equivalents (`--tls`, ...). The problem with that is that they are only supported starting with MongoDB 4.2 which is far above the our stated minimum version (3.6 AFAIK). The downside of using `--ssl` for now is just a warning, although we may have to switch in the future if `--ssl` is ever dropped.

Reply:
For `monosh` this legacy `--ssl` flags generates some UX noice: 
```
tsh db connect --db-user=alice mongo
WARNING: argument --ssl is deprecated and will be removed. Use --tls instead.
WARNING: argument --sslPEMKeyFile is deprecated and will be removed. Use --tlsCertificateKeyFile instead.
Current Mongosh Log ID:	61bb8591520ca2940f8e8483
Connecting to:		mongodb://ice-berg.dev:2345/?directConnection=true
Using MongoDB:		5.0.2
Using Mongosh:		1.0.5
```
So it would be worth to replace repalce them with their modern `--tls`  equivalents.

Reply:
We can use `--tls` with `mongosh` (guaranteed to have it) and keep using `--ssl` when it isn't available. How does that sound?

Reply:
Sound good.